packages/ds-healthcare-gov/src/components/Header/ActionMenu.tsx

```diff
@@ -9,6 +9,7 @@ const menuId = 'hc-c-menu';
 export interface LinkProps {
   href: string;
   label: React.ReactNode;
+  className?: string;
   onClick?: (event: SyntheticEvent) => any;
 }
```

Comment:
This brought to my attention that we can actually use the `Link` type here instead of recreating the type definition. Can import it from `import { Link } from './Header';`